README.adoc

```diff
@@ -131,27 +131,27 @@ the JAX-RS framework.
 ----
 
 [role="edit_command_text"]
-Add OpenAPI annotations to the two JAX-RS methods, [hotspot=32-46 hotspot=48-52 file=0]`getPropertiesForHost()` and [hotspot=70-81 file=0]`listContents()`.
+Add OpenAPI annotations to the two JAX-RS methods, [hotspot=host-property hotspot=Parameter file=0]`getPropertiesForHost()` and [hotspot=listContents file=0]`listContents()`.
 
 
 InventoryResource.java
-[source, Java, linenums, role='code_column']
+[source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[tags=**;!copyright]
+include::finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[]
 ----
 
 Clearly, there are many more annotations now, so let's break them down:
 
 [cols="35, 200", options="header"]
 |===
 | *Annotation*    | *Description*
-| [hotspot=32-42 file=0]`@APIResponses` | A container for multiple responses from an API operation. This annotation is
+| [hotspot=APIResponses file=0]`@APIResponses` | A container for multiple responses from an API operation. This annotation is
```

Comment:
Still highlights more than just the annotation, look at Nimrah's comment.

Reply:
Only highlights one annotation.
<img width="1365" alt="image" src="https://user-images.githubusercontent.com/25497083/68797190-1b9ab580-0622-11ea-8b81-fbda2ac83ccd.png">